src/state/paths.rs

```diff
@@ -0,0 +1,52 @@
+/*
+    Copyright © 2023, ParallelChain Lab
+    Licensed under the Apache License, Version 2.0: http://www.apache.org/licenses/LICENSE-2.0
+*/
+//! This module defines the location of each of the variables stored in the key-value store.
+//! These variables are:
+//! 
+//! |Variable|"Type"|Description|
```

Comment:
This table seems to just be a duplicate of the table in `crate::state::block_tree`. I wonder if it is really necessary to duplicate this table.

Additionally, the links in the second column of this table are malformed.